src/hooks/useLocalNotifications.tsx

```diff
@@ -71,8 +73,19 @@ const useLocalNotificationsContext = () => {
   );
 
   const addTransferNotification = useCallback(
-    (notification: TransferNotifcation) =>
-      setTransferNotifications([...transferNotifications, notification]),
+    (notification: TransferNotifcation) => {
+      const { txHash, triggeredAt, toAmount, type } = notification;
+      setTransferNotifications([...transferNotifications, notification]);
+      // track initialized new transfer notification
+      track(AnalyticsEvent.TransferNotification, {
+        triggeredAt,
+        timeSpent: triggeredAt ? Date.now() - triggeredAt : undefined,
```

Comment:
q: should this just be 0? or there's already a lag here

Reply:
i originally had 0 but i didn't want to hardcode a value since that's technically less accurate. i felt like doing the calculation can be a good indicator if we have a bug (i.e. laggy app, incorrectly rearranging order of functions, etc.)